app/main/posts/views/post-view-map.controller.js

```diff
@@ -0,0 +1,196 @@
+module.exports = ['PostEndpoint', 'Maps', '_', 'PostFilters', 'Leaflet', '$q', '$rootScope', '$compile', '$location', '$timeout', '$scope',
```

Comment:
mostly just moved the map code here because it gives us as bit of flexibility later on and just a smaller directive class which we could eventually separate further especially to have less logic and endpoint calls in . ? Not sure, maybe it was silly. LMK

Reply:
Ah. We want smaller controllers, so that the controllers eventually become components. I would vastly prefer going the other direction -> move code into directive.
If you think the directive should be split we could do that in a follow up. But please make the controller as small as possible

Reply:
yea.. so, I can just revert back to the previous state then. 
The benefit of having a controller, big as it is (which, obviously, is just almost as big as the directive was) is that you can reference the controller functions at any point from a parent which is something we needed at one point because of a weird thing I had done. I guess I can just revert it back now. Will check with you in the afternoon when you are on

Reply:
updated.